pkgs/by-name/ow/owi/package.nix

```diff
@@ -32,9 +32,9 @@ ocamlPackages.buildDunePackage rec {
     llvmPackages.clang-unwrapped
     # lld + llc isn't included in unwrapped, so we pull it in here
     llvmPackages.bintools-unwrapped
+    makeWrapper
     rustc
     zig
-    makeWrapper
```

Comment:
Maybe we could avoid this?  It's unnecessary.

Reply:
The program needs it to have rustc, zig, etc in its path

Reply:
Oh yes. I agree. Wasn't going to nitpick tho.